specification/netapp/resource-manager/Microsoft.NetApp/stable/2019-10-01/netapp.json

```diff
@@ -0,0 +1,2710 @@
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroup"
+          },
+          {
+            "$ref": "#/parameters/AccountName"
+          },
+          {
+            "$ref": "#/parameters/PoolName"
+          },
+          {
+            "$ref": "#/parameters/VolumeName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/replicationStatus"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed."
+          }
+        },
+        "x-ms-long-running-operation": false,
+        "x-ms-examples": {
+          "Volumes_ReplicationStatus": {
+            "$ref": "examples/Volumes_ReplicationStatus.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.NetApp/netAppAccounts/{accountName}/capacityPools/{poolName}/volumes/{volumeName}/resyncReplication": {
+      "post": {
+        "tags": [
+          "Volumes Replication"
+        ],
+        "summary": "Resync volume replication",
+        "description": "Resync the connection on the destination volume. If the operation is ran on the source volume it will reverse-resync the connection and sync from source to destination.",
+        "operationId": "Volumes_ResyncReplication",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroup"
+          },
+          {
+            "$ref": "#/parameters/AccountName"
+          },
+          {
+            "$ref": "#/parameters/PoolName"
+          },
+          {
+            "$ref": "#/parameters/VolumeName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed."
+          }
+        },
+        "x-ms-examples": {
+          "Volumes_ResyncReplication": {
+            "$ref": "examples/Volumes_ResyncReplication.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.NetApp/netAppAccounts/{accountName}/capacityPools/{poolName}/volumes/{volumeName}/deleteReplication": {
+      "post": {
+        "tags": [
+          "Volumes Replication"
+        ],
+        "summary": "Delete volume replication",
+        "description": "Delete the replication connection on the destination volume, and send release to the source replication",
+        "operationId": "Volumes_DeleteReplication",
```

Comment:
How is the replication created initially?  Via "authorize"?  If replication setup is performed via POST users won't be able to setup replication in a template deployment

Reply:
Authorise is a necessary part but only one aspect of setting up a replication. I'm not clear why this being POST would cause issues with a template deployment but I'd agree then this probably needs followed up. We have JIRA tickets, visible to Microsoft, where we track comments during this swagger process that cannot be resolved at the time. It looks like we need to add these points to a new ticket for resolution in the next API.